fxola/Banka-react
# #167116073 Setup Landing page, Sign up and Sign in Templates

#### What does this PR do?
- Setup Landing page, Sign up and Sign in Templates

#### Description of Task to be completed?
- write sign up, sign in and landing page templates
- setup routing between templates
- add svg file and svg loader webpack config
- add fonts
- write tests for templates
- delivers[#167116073]

#### How should this be manually tested?
- Clone the repository using ```git clone https://github.com/fxola/Banka-react.git .``` in your terminal
- run ```npm install``` on your terminal
- run ```npm run start``` to start up the server
- run ```npm test``` to see test results


#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
-  [#167116073](https://www.pivotaltracker.com/n/projects/2362960/stories/167116073)

#### Screenshots (if appropriate)
- N/A

#### Questions:
- N/A
